modules/openapi-generator/src/main/resources/dart-next/libraries/dio/api.mustache

```diff
@@ -0,0 +1,27 @@
+{{>header}}
+import 'dart:async';
+import 'package:dio/dio.dart';
+{{#useBuiltValue}}
+import 'package:built_value/built_value.dart';
+import 'package:built_collection/built_collection.dart';
+import 'package:built_value/serializer.dart';{{/useBuiltValue}}{{#useJsonSerializable}}
+import 'package:json_annotation/json_annotation.dart';{{/useJsonSerializable}}
+import 'package:{{pubName}}/{{sourceFolder}}/repository_base.dart';
+import 'package:{{pubName}}/{{sourceFolder}}/api_util.dart';
+import 'package:{{pubName}}/models.dart';
+{{#operations}}
+{{#imports}}import '{{.}}';
+{{/imports}}
+
+part '{{classFilename}}.g.dart';
```

Comment:
This currently fails when there are no inline enums

Reply:
yes, I am aware of that, and I am working on a fix rn and completing the generator changes

Reply:
Nice, ill see if I have time to test during the FlutterCon days.

Reply:
@ahmednfwela Do you have a fix for this?

Reply:
@kuhnroyal yes, but I am a little busy for a couple of days

Reply:
I am pushing a fix for it today , sorry for the delay